dataservice/api/common/schemas.py

```diff
@@ -156,6 +154,39 @@ class PaginatedSchema(Schema):
     return PaginatedSchema
 
 
+def filter_schema_factory(model_filter_schema_cls):
```

Comment:
Why not take the base schema and construct the filter schema here dynamically? That way you don't need to create an empty `*FilterSchema` for everything.

Reply:
Well I wasn't sure exactly how to dynamically define classes otherwise yea thats exactly what I would want to do. Let me look into it

Reply:
I think it would be something like `FilterClass = type('MyFilterClass', (base_cls, FilterSchemaMixin), {})`